rwengine/src/audio/SoundManager.hpp

```diff
@@ -117,6 +123,9 @@ class SoundManager {
 
     /// Multiplier for music and cutscenes audio
     float _musicVolume = 1.f;
+
+    /// Available sound effects for sfx
+    std::vector<std::shared_ptr<SoundEffect>> soundEffects;
```

Comment:
`std::vector<std::unique_ptr<SoundEffect>> soundEffects;` should be enough. (pass raw ptr to `Sound`s)